devdocs/settingUp.md

```diff
@@ -209,6 +209,14 @@ TBD
     deployed app and not the dev server).
     Note that GAE daily quota will be exhausted after 2-3 runs of the full test suite.
 
+
+##Run client scripts
+Client scripts are scripts that help us remotely manipulate data on GAE via its Remote API. Most of developers may not need to write and/or run client scripts but if you are to do so, additional steps are required:
+1. Download and install Google Cloud SDK at https://cloud.google.com/sdk/downloads.
```

Comment:
The numbered list isn't getting rendered correctly, probably need to have a newline before it.